### PR TITLE
pretty formatting

### PR DESCRIPTION
**Pull request by magicse**
changed size of images for pretty formatting of page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -308,9 +308,8 @@ ncnn 目前已在腾讯多款应用中使用，如：QQ，Qzone，微信，天
...
-<img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/ncnn-33.jpg" width="360" height="640"/><img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/ncnn-m.png" width="360" height="640"/>
-<img src="https://github.com/nihui/ncnn-android-yolov5/raw/master/screenshot.jpg" width="360" height="800"/>功能概述<img src="https://github.com/nihui/ncnn-android-scrfd/raw/master/screenshot.jpg" width="360" height="800"/>
+<img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/ncnn-2.jpg" height ="270"/><img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/4.jpg" height ="270"/><img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/ncnn-33.jpg" height ="270"/><img src="https://github.com/nihui/ncnn-assets/raw/master/20181217/ncnn-m.png" height ="270"/><img src="https://github.com/nihui/ncnn-android-yolov5/raw/master/screenshot.jpg" height ="270"/><img src="https://github.com/nihui/ncnn-android-scrfd/raw/master/screenshot.jpg" height ="270"/><br>
+功能概述
```
Comment (nihui) [inline README.md:312]: messed text ?

**Review by nihui — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (magicse) [reply README.md:312]: Hi Nihui. Images was so big and not beauty formatted before. 
Now images will be in one line.

Comment (nihui) [reply README.md:312]: yeah, but '功能概述' is redundant here

Comment (magicse) [reply README.md:312]: Yes sure it could be removed

**Review by nihui — APPROVED** [review]

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**